daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ComparisonSBuiltinTest.scala

```diff
@@ -114,7 +114,9 @@ class ComparisonSBuiltinTest extends AnyWordSpec with Matchers with TableDrivenP
             e"1970-01-01T00:00:00.000000Z",
             e"2020-02-02T20:20:02.020000Z",
           ),
-          t"Party" -> List(e"'alice'", e"'bob'", e"'carol'"),
+          // Parties cannot be built from expressions.
+          // We map at runtime the `party1`, `party2` and, `party3` two 3 party IDS in increasing order.
+          t"Party" -> List(e"party1", e"party2", e"party3"),
```

Comment:
```suggestion
           t"Party" -> List(
            e"""TEXT_TO_PARTY "alice"""",
            e"""TEXT_TO_PARTY "bob"""",
            e"""TEXT_TO_PARTY "carol"""",
          ),
```

Reply:
I’ve reverted that again, The fact that it returns an `Optional` (which works but is against the spirit of this test imho) and that I’d like to eventually kill that primitive completely makes me dislike that approach. Happy to discuss if you disagree.